src/argparser.py

```diff
@@ -389,8 +410,8 @@ def omniarg_parser(parser, omniperf_home, omniperf_version):
     analyze_group.add_argument(
         "--list-metrics",
         metavar="",
-        choices=["gfx906", "gfx908", "gfx90a"],
-        help="\t\tList all available metrics for analysis on specified arch:\n\t\t   gfx906\n\t\t   gfx908\n\t\t   gfx90a",
+        choices=load_avail_arch(omniperf_home),#["gfx906", "gfx908", "gfx90a"],
+        help=print_avail_arch(load_avail_arch(omniperf_home)),
```

Comment:
Thanks for this, I know I suggested we move away from the hard-coded list but not sure about the replacement based on file parsing.